<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>901</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout1">
       <item row="0" column="0">
        <widget class="QGroupBox" name="gBoxFieldsValue">
         <property name="title">
          <string>Fields Values to Calculate...</string>
         </property>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>861</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lbFirstValue">
             <property name="text">
              <string>First Value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEFirstValue">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbSecondValue">
             <property name="text">
              <string>Second Value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lESecondValue">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="gBoxActions">
         <property name="title">
          <string>Action Buttons...</string>
         </property>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>861</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pButtonAdd">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pButtonSub">
             <property name="text">
              <string>Sub</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pButtonMul">
             <property name="text">
              <string>Mul</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pButtonDiv">
             <property name="text">
              <string>Div</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="gBoxResult">
         <property name="title">
          <string>Result Calc...</string>
         </property>
         <widget class="QLineEdit" name="lEResult">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>70</y>
            <width>291</width>
            <height>71</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
